### PR TITLE
Handle PENDING status for launched scan but still not started by ospd.

### PR DESCRIPTION
**Pull request by jjnicola**
Depends on PR greenbone/gvm-libs#336

### CODE REVIEW

File: src/manage.c [hunk 1]
```diff
@@ -3630,7 +3630,8 @@ handle_osp_scan (task_t task, report_t report, const char *scan_id)
                   rc = 0;
                   break;
                 }
-              else if (osp_scan_status == OSP_SCAN_STATUS_RUNNING
+              else if ((osp_scan_status == OSP_SCAN_STATUS_RUNNING
+                        || osp_scan_status == OSP_SCAN_STATUS_RUNNING)
```
Comment (mattmundell) [inline src/manage.c:3634]:
```suggestion
                        || osp_scan_status == OSP_SCAN_STATUS_PENDING)
```

Comment (jjnicola) [reply src/manage.c]: :open_mouth:

**Comment by mattmundell** [comment]
Also, there's another case of OSP_SCAN_STATUS_RUNNING in prepare_osp_scan_for_resume.  Is that relevant?

**Comment by jjnicola** [comment]
PENDING status does not need to be handled. It is the same case as for INIT, which is ignored as well.

**Pull request closed without merging**